nx.json

```diff
@@ -13,27 +13,19 @@
     }
   },
   "release": {
+    "projects": ["rxjs"],
     "releaseTagPattern": "{version}",
     "changelog": {
-      "git": {
```

Comment:
Committing and tagging by default after/as part of the changelog generation step just makes a lot sense based on more and more real world usage, so we updated the latest nx to make it so and can streamline the config. As before the logs, in both dry-run and non dry-run, explain when and what git commands are/would be running